build/templates/session.py.mako

```diff
@@ -191,9 +191,9 @@ class Session(_SessionBase):
     def close(self):
         try:
             self._close()
-        except errors.Error:
-            # TODO(marcoskirsch): This will occur when session is "stolen". Change to log instead
-            print("Failed to close session.")
+        except errors.Error as e:
+            self._${config['session_handle_parameter_name']} = 0
+            raise errors.Error(e.code, e.description)
```

Comment:
Use raise without any parameters and it will rethrow the original exception

Reply:
Fixed!